flink-runtime/src/main/java/org/apache/flink/runtime/checkpoint/CheckpointCoordinator.java

```diff
@@ -43,7 +43,6 @@
 import org.apache.flink.runtime.state.StateBackend;
```

Comment:
Generally speaking I'm not a big fan of this last commit, as it will make our lives more difficult to re-apply reverted refactoring commits. So I would actually prefer to drop/suspend it for now.

Reply:
As discussed offline, let me know if I should drop it (or feel free to drop it yourself).